std/algorithm/setops.d

```diff
@@ -870,14 +941,17 @@ as a range of ranges and each is assumed to be sorted by $(D
 less). Computation is done lazily, one union element at a time.
 `multiwayUnion(ror)` is functionally equivalent to `multiwayMerge(ror).uniq`.
 
+`multiwayUnion` returns a set regardless of the type of the input ranges
+(sets or multisets).
```

Comment:
We don't distinguish sets/multisets by "type" so the sentence is misleading. Replace with:

"The output of `multiwayUnion` has no duplicates even when its inputs contain duplicates."